source/includes/APIReference/Jobcodes/Examples/_create-request.md.erb

```diff
@@ -7,15 +7,17 @@
      "short_code":"c1",
      "billable":"yes",
      "assigned_to_all":"yes",
-     "billable_rate":"37.50"
+     "billable_rate":"37.50",
+     "connect_with_quickbooks":true
     },
     {
      "name":"Customer2",
      "parent_id":"12788279",
      "short_code":"c1",
      "billable":"no",
      "assigned_to_all":"no",
-     "billable_rate":"37.50"
+     "billable_rate":"37.50",
+     "connect_with_quickbooks":false
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
@dustinsweet Is it OK that there is a newline at the end of the file? My IDE is setup to add these on save which is required for PHP.